core/src/main/java/net/adoptopenjdk/icedteaweb/resources/ResourceTracker.java

```diff
@@ -388,20 +384,29 @@ private Resource getResource(URL location) {
      * @throws InterruptedException if another thread interrupted the wait
      */
     private void wait(Resource... resources) throws InterruptedException {
-        // save futures in list to allow parallel start of all resources
-        final List<Future<Resource>> futures = Stream.of(resources)
-                .map(ResourceHandler::new)
-                .map(ResourceHandler::putIntoCache)
+        wait(resources, 100_000_000, TimeUnit.DAYS);
+    }
+
+    public static final String getSimpleName(Resource resource) {
```

Comment:
This method should be in the class `Resource`

Reply:
... or removed once I'm done ;) This just helped to create a lot of log.l Way to many log messages since we do not have trace logging